test/system/admin/article_test.rb

```diff
@@ -14,7 +14,7 @@ class AdminArticleTest < ApplicationSystemTestCase
     @partner = create :partner
     @partner_two = create :ashton_partner
 
-    @tag = create :tag
+    @tag = create(:tag, type: 'Partnership')
```

Comment:
Do we have a `:partnership` factory?

Reply:
so in the [tag factory](https://github.com/geeksforsocialchange/PlaceCal/blob/2061-tag_admin-to-partnership_admin/test/factories/tag.rb)

you want to add another scope like
```ruby
    factory :partnership do
      sequence(:name) { |n| "Partnership Tag #{n}" }
      type { 'Partnership' }
    end
```

Reply:
I got a but thrown by the STI thing but this makes lots more sense thinking about how we do Users